### ISSUE TITLE
Initial Classification

### ISSUE THREAD

**Issue by ozaltun** [lamaoudi/883_idtrees#3]
Once issues #1 and #2 are finalized and the dataset is prepared and we have a training dataset, we will need to run classification algorithms.

The goal of this issue is to report initial results using out of the box classification functions to get accuracy measures.

**Comment by ozaltun** [comment]
@lamaoudi related to my comment in #1, I have added some classification results to the new notebook:
1. An updated SVM (exact code you use), but replacing the data with balanced data and removing Other. The confusion matrix looks better to me, thoughts? The higher accuracy is mostly due to removing Other.
2. I also just ran a bunch of other out-of-the-box algorithms, RandomForestClassifier seems to be doing great! XGBoost is still running, will update the notebook in East Coast morning.

**Next step thoughts:**
Let me know your thoughts/comments. Maybe we want to finish the document write-up and then play around with some additional configurations this week and submit my Friday. How does that sound to you?

**Comment by lamaoudi** [comment]
Sounds like a plan. I'll also add that I want to play around with the PCA decomposition. Right now it only reduces the data by 9 bands, I expect classification algorithms to perform better if we further reduce the number of features they need to learn?

**Comment by ozaltun** [comment]
@lamaoudi added some results to the `BO` notebook. Additionally, outputted the 5-fold CV for the estimations in the [output](https://github.com/lamaoudi/883_idtrees/blob/master/output/model_runs_BO.csv) folder. XGBoost and Random Forest seem to be performing the best. Let me know if/when you update the PCAs and I can go ahead and re-run.

Another note on the PCA, if you want we can try running with a couple different reductions to see the sensitivity of estimation to the reduction.
